Subaharish/HTML BASICS/assignment3.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Registration</title>
+    </head>
+    <body>
+        <h1>Register</h1>
+        <form >
+            First name:
+            <input type="text" name="Firstname"  placeholder="Firstname" required autocomplete="family-name">
+            Last name:
+            <input type="text" name="Lastname" placeholder="Lastname" required autocomplete="name">
```

Comment:
Wrong `autocomplete` attributes. Refer the site I gave again: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)